server/index.js

```diff
@@ -34,6 +35,12 @@ function main() {
     this.set('Timing-Allow-Origin', '*');
     yield next;
   });
+
+
+  app.use(cors({
```

Comment:
But it's also needed for the stuff in public – `/js/liveblog.js`, `/css/pages/liveblog.css`